src/styles/card.scss

```diff
@@ -0,0 +1,128 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  gap: 30px 48px;
+  font-family: Roboto, sans-serif;
+
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    width: 200px;
```

Comment:
The padding and width here can be extracted to variables for better maintainability, especially since the width is reused in the grid-template-columns definition.